### ISSUE TITLE
Missing localization settings in Spreadsheet

### ISSUE THREAD

**Issue by nenchef** [telerik/kendo-ui-core#3981]
### Bug report
Missing ability to manage the following localization settings in Spreadsheet

* Right click menu items on cells, column and row
* 'All' in the filter by value menu
* Default sheet name when inserting a new sheet
* Element type in PDF export
* Margin type selection in PDf export
* Paper type in pdf export
* Default document name in export (pdf | xls)
* Hyperlink dialog title message

### Reproduction of the problem
Example for Hyperlink dialog title message - apply the following configuration to Spreadsheet:
```
$.extend(true, kendo.spreadsheet.messages.dialogs,{
      "linkDialog": {
          "title": "Hyperlink my dialog",
          "labels": {
              "text": "Text",
              "url": "My Address",
              "removeLink": "Remove my link"
          }
      }
    });
```
[Dojo](http://dojo.telerik.com/AniTUrIx)

### Current behavior
The dialog title is not changed. (URL and removeLink are applied as expected)

### Expected/desired behavior
The dialog title is changed to "Hyperlink my dialog".

### Environment
* **Kendo UI version:** 2018.2.620
* **Browser:** [all]

**Comment by mishoo** [comment]
The following new messages have been added (branch `spreadsheet-messages`):

```js
kendo.spreadsheet.messages: {
  dialogs: {
    exportAsDialog: {
      defaultFileName: "Workbook",
      xlsx: {
        description: "Excel Workbook (.xlsx)"
      },
      pdf: {
        description: "Portable Document Format (.pdf)",
        area: {
          workbook: "Entire Workbook",
          sheet: "Active Sheet",
          selection: "Selection"
        },
        paper: {
          a2        : "A2 (420 mm × 594 mm)",
          a3        : "A3 (297 mm x 420 mm)",
          a4        : "A4 (210 mm x 297 mm)",
          a5        : "A5 (148 mm x 210 mm)",
          b3        : "B3 (353 mm × 500 mm)",
          b4        : "B4 (250 mm x 353 mm)",
          b5        : "B5 (176 mm x 250 mm)",
          folio     : 'Folio (8.5" x 13")',
          legal     : 'Legal (8.5" x 14")',
          letter    : 'Letter (8.5" x 11")',
          tabloid   : 'Tabloid (11" x 17")',
          executive : 'Executive (7.25" x 10.5")'
        },
        margin: {
          normal: "Normal",
          narrow: "Narrow",
          wide: "Wide"
        }
      },
    }
  },
  filterMenu: {
    all: "All"
  },
  menus = {
    cut     : "Cut",
    copy    : "Copy",
    paste   : "Paste",
    merge   : "Merge",
    unmerge : "Unmerge",
    delete  : "Delete",
    hide    : "Hide",
    unhide  : "Unhide"
  },
  workbook: {
    defaultSheetName: "Sheet"
  }
}
```
